kaolin/datasets/shapenet.py

```diff
@@ -193,6 +160,85 @@ def __getitem__(self, index):
         return {'data': data, 'attributes': attributes}
 
 
+class ShapeNet(KaolinDataset):
+    r"""ShapeNet Dataset class for meshes.
+    Args:
+        root (str): path to ShapeNet root directory
+        categories (list): List of categories to load from ShapeNet. This list may
+                           contain synset ids, class label names (for ShapeNetCore classes),
+                           or a combination of both.
+        train (bool): is using training split (default: True)
+        split (float): proportion of the dataset to split to the training datase
+    Returns:
+        .. code-block::
```

Comment:
Don't we need a blank line below `.. code-block::` in order for it to render properly when built?

Reply:
Good catch, fixed (and I also fixed indentation)